homeassistant/components/elevenlabs/quality_scale.yaml

```diff
@@ -0,0 +1,85 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: >
+      Only entity services
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: todo
+  runtime-data: done
```

Comment:
`EleventLabsConfigEntry` typo

Reply:
Agreed, but already on dev. Would you rather have me fix it in this PR or in a separate one?

Reply:
Separate one